firstvoices/backend/views/search_all_entries_views.py

```diff
@@ -574,13 +594,17 @@ def get_search_params(self):
         sort = self.request.GET.get("sort", "")
         valid_sort, descending = get_valid_sort(sort)
 
+        sites = self.request.GET.get("sites", "")
+        valid_site_ids = get_valid_site_ids_from_slugs(sites, user)
+
         search_params = {
             "q": input_q,
             "user": user,
             "types": valid_types_list,
             "domain": valid_domain,
             "kids": kids_flag,
             "games": games_flag,
+            "sites": valid_site_ids,
             "site_id": "",  # used in site-search
```

Comment:
I'm looking at these two parameters-- sites and site_id. What if we combine them? The rest APIs would stay the same, but this function could just use the sites list. For the site-specific search, we could put the single site_id into the sites list parameter. It would avoid any weird scenarios where we accidentally try to use both at the same time